### PR TITLE
Web3 Login + Home page + Header

### PR DESCRIPTION
**Pull request by johnrjj**
Ready for review

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+
+  useInactiveListener(!hasTriedEagerConnect);
+
+  return (
+    <>
+      <HeaderWrapperOuter>
+        <HeaderWrapperInner>
+          <LeftWrapper>
+            <Link to={'/'}>
+              <Logo />
```
Comment (CarbonEdge) [inline src/components/Header.tsx:196]: I think you are getting console errors from nesting a link tag like this

Warning: validateDOMNesting(...): <a> cannot appear as a descendant of <a>.

Comment (CarbonEdge) [reply src/components/Header.tsx:196]: nice page though

Comment (ape-dev-cs) [reply src/components/Header.tsx:196]: I've fixed this up, I think we can merge now.

**Comment by ape-dev-cs** [comment]
Have read over the code & everything looks good to me. Really nice stuff!

**Pull request merged**